ROVU/Core/src/model/RewardCalculator.java

```diff
@@ -1,13 +1,11 @@
 package model;
 
-import controller.DataHandler;
-import controller.RobotController;
```

Comment:
The `DataHandler` and the `RobotController` needs to stay there, the usage has been commented out in the meanwhile, but it's for future usage.

Reply:
Done, added a `@SuppressWarnings("unused")` instead.